src/test/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentControllerTest.java

```diff
@@ -20,16 +20,17 @@
 import uk.gov.companieshouse.documentsigningapi.signing.SigningService;
 import uk.gov.companieshouse.logging.Logger;
 
+import javax.servlet.http.HttpServletRequest;
 import java.net.URISyntaxException;
 import java.util.List;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.lenient;
+import static uk.gov.companieshouse.documentsigningapi.util.TestConstants.*;
```

Comment:
Its best practice to not use the wildcard in imports and instead import only what you need. It may be a setting in you IDE .